server/server.go

```diff
@@ -214,18 +228,25 @@ func (s *Server) handle(conn *gokeyless.Conn) {
 		if err != nil {
 			s.Log.Printf("%s: Signing error: %v\n", gokeyless.ErrCrypto, err)
 			connError = conn.RespondError(h.ID, gokeyless.ErrCrypto)
+			s.stats.logInvalid(requestBegin)
 			continue
 		}
 
 		connError = conn.Respond(h.ID, sig)
+		s.stats.logRequest(requestBegin)
 	}
+
 	s.Log.Printf("Connection error: %v\n", connError)
-	return
+}
+
+func (s *Server) serveMetrics() {
```

Comment:
Both good points. Because I didn't clean them up when I separated out the two files. *fixed*